### ISSUE TITLE
FEATURE: Game-Master Tools

### ISSUE THREAD

**Issue by jkhoel** [132nd-vWing/OPBR-Brief#4]
I think we would need a way to "guide" the AI, maybe in-between sessions, so that we can nudge them into doing something. Say attack Khazab, so that we can recapture it next game-night (Weekly Objective).

**Comment by 132ndNeck** [comment]
What are the options and limitations for this?

And how can it be handled?

Should be based on giving a larger "formation" of enemy units (regiment, etc) that is tied together somehow, and when this parent unit gets this task, then all sub units/groups act accordingly, or start moving into position.

We could then have 2-6 such areas or lines of communication where the battle takes place, and have 1 active at any given time (to reduced load on the server), and static units in the areas where not much is happening.  

Also ties to Issue #2 , where after an offensive, the unit gets static, and need resupply by some sort of logistical convoy, and reinforced for losses. This must take place before it can go on the offensive again.


Also this game master tools can/shoud? have the option for initiation such a logistical resupply.

So to sum it up , need possibility for:
- Go on the offensive
- Stay in defensive positions (static)
- Stay in defence positions (active units) - expected battle with blue forces
- Initiate logistical resupply
- Withdrawal/retreat operations

Any other?

**Comment by thebgpikester** [comment]
we use two common ones. Spawn "thing" via map marker and Delete all things in radius.

Very handy. have code for that. rest of it is just moving as per normal GM.
